release/config.ini.sample

```diff
@@ -252,6 +252,21 @@ packetThreads=2
 # and includes subdomains (the app must be served over https)
 #hstsHeader=true
 
+# Business Hours
+# If set, displays a colored background on the sessions timeline graph during business hours
+# Business hours are assumed to be on a daily basis Monday through Friday
+# Values are set in hours from midnight (default is off)
+#businessDayStart=9
+#businessDayEnd=17
+
+# Business Days
+# Comma separated list of days
+# If set, displays the business hours on only the days provided here
+# Business hours must be set for these to be of use
+# Values are the days of the week as numbers, the week starts at Sunday = 0 and ends on Saturday = 6
+# (default is Monday - Friday 1,2,3,4,5)
```

Comment:
@awick good default?

Reply:
yep, the only other common weekends is fri/sat